### PR TITLE
[DOCS-41] Added Rider section

### PR DESCRIPTION
**Pull request by warrantor**

### CODE REVIEW

File: package-managers/nuget.md [hunk 1]
```diff
@@ -54,36 +54,64 @@ inspiration: http://docs.myget.org/docs/reference/credential-provider-for-visual
...
+
+In order to consume our private feed we have to give the feed URL and enter our credentials.
+
+![Feed with credentials](assets/feed-with-credentials.png)
+
+While Rider supports using pre-authenticated feeds as well as feeds that require entering credentials, we recommend using the latter. Rider safely stores our MyGet username/password in its [password store](https://www.jetbrains.com/help/idea/passwords.html), which is based on [KeePass](http://keepass.info/).
+
+#### Public feed
+
+To work with public feeds you just leave the credentials empty and Rider will connect to the feed.
```
Comment (maartenba) [inline package-managers/nuget.md]: `To work with public feeds, enter the feed URL and leave the credentials empty. Rider will then connect to the feed.``

File: package-managers/nuget.md [hunk 2]
```diff
@@ -54,36 +54,64 @@ inspiration: http://docs.myget.org/docs/reference/credential-provider-for-visual
...
+
+![Feed without credentials](assets/rider-feed-no-credentials.png)
+
+<p class="alert alert-error">
+    <strong><TODO: change this to a Windows dialog/strong>
```
Comment (maartenba) [inline package-managers/nuget.md]: Eh wat? :-)

**Review by maartenba — APPROVED** [review]
Almost there (the TODO still?)

File: package-managers/nuget.md (hunk 1 above)
Comment (warrantor) [reply package-managers/nuget.md]: Maybe better. 😝

File: package-managers/nuget.md (hunk 2 above)
Comment (warrantor) [reply package-managers/nuget.md]: Had geen zin om vandaag mijn Windows machine nog op te straten voor ne screenshot. Moet die dus nog aanpassen.

File: package-managers/nuget.md [hunk 3]
```diff
@@ -54,36 +54,69 @@ inspiration: http://docs.myget.org/docs/reference/credential-provider-for-visual
...
-To find the feed URL check the [Find our feed URL](#find-our-feed-url) section.
+![Nuget tool window](assets/rider-feeds-pane.png)
+
+From here, we can add our MyGet feed (or edit an existing entry). We will have to give our feed a name so we can easily recognize it in Rider, and the URL to our feed.
 
 #### Private feed
 
+Details on how to find the feed URL can be found in the [Find the feed URL](#find-the-feed-url) section.
+
+In order to consume our private feed we have to give the feed URL and enter our credentials.
```
Comment (maartenba) [inline package-managers/nuget.md:67]: `give` -> `provide`

File: package-managers/nuget.md [hunk 4]
```diff
@@ -54,36 +54,69 @@ inspiration: http://docs.myget.org/docs/reference/credential-provider-for-visual
...
-To find the feed URL check the [Find our feed URL](#find-our-feed-url) section.
+Details on how to find the feed URL can be found in the [Find the feed URL](#find-the-feed-url) section.
 
 #### Private feed
 
+Details on how to find the feed URL can be found in the [Find the feed URL](#find-the-feed-url) section.
+
 ## Creating NuGet/.NET Core packages
 
 could have links to the NuGet documentation
```
Comment (maartenba) [inline package-managers/nuget.md:122]: This is one of those TODO's ;-)

**Review by maartenba — COMMENTED** [review]
lgtm, minor nit

File: package-managers/nuget.md (hunk 3 above)
Comment (warrantor) [reply package-managers/nuget.md:67]: Done

File: package-managers/nuget.md (hunk 4 above)
Comment (warrantor) [reply package-managers/nuget.md:122]: As in, ...

File: package-managers/nuget.md [hunk 5]
```diff
@@ -54,7 +54,7 @@ inspiration: http://docs.myget.org/docs/reference/credential-provider-for-visual
 
 ### JetBrains Rider
 
-The first step in connecting Rider to a MyGet feed is adding it as a package source. We can do this using NuGet.exe (via good old [NuGet.config](https://docs.microsoft.com/en-us/nuget/schema/nuget-config-file#package-source-sections), or from within Rider. From the **NuGet tool window**, open the **Sources** tab. This will show us all of the NuGet configuration files that are in play, and a list of all feeds configured.
+The first step in connecting Rider to a MyGet feed is adding it as a package source. We can do this using NuGet.exe (via good old [NuGet.config](https://docs.microsoft.com/en-us/nuget/schema/nuget-config-file#package-source-sections), or from within Rider. From the **NuGet tool window** (<kbd>Alt</kbd> + <kbd>7</kbd> will bring this right up), open the **Sources** tab. This will show us all of the NuGet configuration files that are in play, and a list of all feeds configured.
```
Comment (maartenba) [inline package-managers/nuget.md]: Drop 'will bring this right upcoming and the . right after. Other than that lgtm

File: package-managers/nuget.md [hunk 6]
```diff
@@ -54,36 +54,69 @@ inspiration: http://docs.myget.org/docs/reference/credential-provider-for-visual
 
 ### JetBrains Rider
 
-#### Public feed
+The first step in connecting Rider to a MyGet feed is adding it as a package source. We can do this using NuGet.exe (via good old [NuGet.config](https://docs.microsoft.com/en-us/nuget/schema/nuget-config-file#package-source-sections), or from within Rider. From the **NuGet tool window** (<kbd>Alt</kbd> + <kbd>7</kbd> will bring this right up), open the **Sources** tab. This will show us all of the NuGet configuration files that are in play, and a list of all feeds configured.
```
Comment (xavierdecoster) [inline package-managers/nuget.md]: Perhaps use back-ticks around `nuget.exe`? (didn't we settle on lowercase `nuget` when referring to the CLI?)

File: package-managers/nuget.md [hunk 7]
```diff
@@ -54,36 +54,69 @@ inspiration: http://docs.myget.org/docs/reference/credential-provider-for-visual
 
 ### JetBrains Rider
 
-#### Public feed
+The first step in connecting Rider to a MyGet feed is adding it as a package source. We can do this using NuGet.exe (via good old [NuGet.config](https://docs.microsoft.com/en-us/nuget/schema/nuget-config-file#package-source-sections), or from within Rider. From the **NuGet tool window** (<kbd>Alt</kbd> + <kbd>7</kbd> will bring this right up), open the **Sources** tab. This will show us all of the NuGet configuration files that are in play, and a list of all feeds configured.
 
-To find the feed URL check the [Find our feed URL](#find-our-feed-url) section.
+![Nuget tool window](assets/rider-feeds-pane.png)
```
Comment (xavierdecoster) [inline package-managers/nuget.md]: nit: Nu**G**et tool window

File: package-managers/nuget.md [hunk 8]
```diff
@@ -54,36 +54,69 @@ inspiration: http://docs.myget.org/docs/reference/credential-provider-for-visual
...
 
+Details on how to find the feed URL can be found in the [Find the feed URL](#find-the-feed-url) section.
+
+In order to consume our private feed we have to give the feed URL and enter our credentials.
+
+![Feed with credentials](assets/feed-with-credentials.png)
+
+While Rider supports using pre-authenticated feeds as well as feeds that require entering credentials, we recommend using the latter. Rider safely stores our MyGet username/password in its [password store](https://www.jetbrains.com/help/idea/passwords.html).
+
+When using the pre-authenticated feed we, of cource, don't need to enter our credentials.
```
Comment (xavierdecoster) [inline package-managers/nuget.md]: cource > course

Comment (xavierdecoster) [reply package-managers/nuget.md]: (missing word) of course, **we** don't need to ...

**Review by xavierdecoster — COMMENTED** [review]
Couple of nits :)

File: package-managers/nuget.md (hunk 6 above)
Comment (maartenba) [reply package-managers/nuget.md]: `NuGet.exe` I believe

File: package-managers/nuget.md (hunk 8 above)
Comment (warrantor) [reply package-managers/nuget.md]: the _we_ is in front of the _of course_

Comment (xavierdecoster) [reply package-managers/nuget.md]: oops, overlooked :)

**Review by xavierdecoster — APPROVED** [review]
:shipit:

**Pull request merged**